### PR TITLE
fix(db_stats.py): Reorganize Prometheus query methods

### PR DESCRIPTION
**Pull request by yarongilor**
Prometheus queries related methods were not organized
	and were located in a specif test and not a common related file.

Trello: https://trello.com/c/OvVNXBMi
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
Tested in: https://jenkins.scylladb.com/job/scylla-staging/job/longevity-tests-ICS/job/features-ics-space-amplification-goal-test-refactor/2/

File: sdcm/db_stats.py [hunk 1]
```diff
@@ -792,3 +801,65 @@ def get_doc_data(self, key) -> Optional[dict]:
...
+    return used_size_gb
+
+
+def get_filesystem_total_size_gb(prometheus_db, node) -> float:
+    """
+    :param prometheus_db:
+    :param node:
+    :return: FS size in GB
+    """
+    fs_size_metric = 'node_filesystem_size_bytes'
```
Comment (fruch) [inline sdcm/db_stats.py:850]: Why it's not using FS_SIZE_METRIC ?

File: sdcm/db_stats.py [hunk 2]
```diff
@@ -792,3 +801,65 @@ def get_doc_data(self, key) -> Optional[dict]:
 
     def get_test_start_time(self):
         return self._stats['test_details'].get('start_time', None)
+
+
+def generate_node_capacity_query_postfix(node):
+    return f'{{mountpoint="{SCYLLA_DIR}", instance=~".*?{node.private_ip_address}.*?"}}'
+
+
+def get_used_capacity_gb(prometheus_db, node):  # pylint: disable=too-many-locals
```
Comment (fruch) [inline sdcm/db_stats.py:810]: Why those aren't part of the PrometheusDBStats, like all the other ones defined in this file ?

**Comment by fruch** [comment]
Is there a test needs to be written, that gonna use those moved functions ? or it's just a leftover comment from when this code was merged?

File: sdcm/db_stats.py [hunk 3]
```diff
@@ -792,3 +801,65 @@ def get_doc_data(self, key) -> Optional[dict]:
...
+
+
+def get_filesystem_total_size_gb(prometheus_db, node) -> float:
+    """
+    :param prometheus_db:
+    :param node:
+    :return: FS size in GB
+    """
+    fs_size_metric = 'node_filesystem_size_bytes'
+    fs_size_metric_old = 'node_filesystem_size'
```
Comment (aleksbykov) [inline sdcm/db_stats.py:851]: Also for FS_SIZE_METRIC_OLD

**Review by aleksbykov — COMMENTED** [review]
May be better move all such fucntion to new file with query methods only?

File: sdcm/db_stats.py (hunk 3 above)
Comment (yarongilor) [reply sdcm/db_stats.py:851]: fixing FS_SIZE_METRIC_OLD

**Comment by yarongilor** [comment]
> Is there a test needs to be written, that gonna use those moved functions ? or it's just a leftover comment from when this code was merged?

yes, test_ics_space_amplification_goal

File: sdcm/db_stats.py (hunk 2 above)
Comment (yarongilor) [reply sdcm/db_stats.py]: fixed

**Comment by yarongilor** [comment]
Tested in: https://jenkins.scylladb.com/job/scylla-staging/job/longevity-tests-ICS/job/features-ics-space-amplification-goal-test-refactor/3/

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**